images/kubectl-build-deploy-dind/build-deploy-docker-compose.sh

```diff
@@ -544,6 +556,9 @@ else
         --set tls_acme="${ROUTE_TLS_ACME}" \
         --set insecure="${ROUTE_INSECURE}" \
         --set hsts="${ROUTE_HSTS}" \
+        --set ingressmonitorcontroller.enabled="${MONITORING_ENABLED}" \
+        --set ingressmonitorcontroller.path="${MONITORING_PATH}" \
+        --set ingressmonitorcontroller.alertContacts="${ALERTCONTACT}" \
```

Comment:
mhh let's agree to use `ALERTCONTACT` or `MONITORING_ALERTCONTACT` it uses both currently

Reply:
yeah, mind split from working on this over such a length of time, standardized on `MONITORING_ALERTCONTACT`